ompi/datatype/ompi_datatype.h

```diff
@@ -150,7 +150,7 @@ ompi_datatype_is_predefined( const ompi_datatype_t* type )
 }
 
 static inline int32_t
-ompi_datatype_is_contiguous_memory_layout( const ompi_datatype_t* type, int32_t count )
+ompi_datatype_is_contiguous_memory_layout( const ompi_datatype_t* type, size_t count )
```

Comment:
This makes no sense without modifying first `opal_datatype_is_contiguous_memory_layout`.